src/index.html

```diff
@@ -7,10 +7,52 @@
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Search bar airbnb</title>
+    <title>Search bar Airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <div class="search-bar__wrapper">
+      <form
+        action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+        class="search-bar__form"
+      >
+        <label for="searchBarBig" class="search-bar__label big" data-qa="big">
```

Comment:
go through [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) once again, pay attention to 3rd item

Reply:
```suggestion
        <label 
         for="searchBarBig"
         class="search-bar__label big"
         data-qa="big"
       >
```

Reply:
Sorryyyy ):